int64 duty_cycle
int32 channel
---
int64 error